docs/how-to/import-prisma-schema.md

```diff
@@ -0,0 +1,120 @@
+---
+id: import-prisma-schema
+title: Import Prisma Schema
+sidebar_label: Import Prisma Schema
+slug: /how-to/import-prisma-schema
+pagination_next: getting-started/how-prisma-schema-is-converted-into-entities
+---
+
+# Import Prisma Schema
+
+Amplication's **Import Prisma Schema** feature lets you use your _existing_ databases for a head start on development.
+
+This way you don't have to manually create entities, fields, and relationships from scratch saving you a lot of time.
+
+Just upload your `schema.prisma` file.
+Amplication will then search for models and [convert them into Amplication's internal structure](/getting-started/how-prisma-schema-is-converted-into-entities).
+
+This guide walks you through how to generate a custom `schema.prisma` file from your existing database and upload it to Amplication.
+
+## How to create and upload your `schema.prisma` file
+
+First, you will run the Prisma introspection process on your existing database.
+The introspection process will generate a schema file representing your database's tables and columns.
+
+Next, you'll upload the generated file into Amplication, ensuring it's free of errors before doing so.
+After upload, you'll see an upload log of everything that happens to [convert the schema into Amplication's internal structure](/getting-started/how-prisma-schema-is-converted-into-entities).
+If any warnings appear in the upload log you can then choose to modify your `schema.prisma` file and re-upload it or continue as is.
+
+### Step 1: Run the Introspection process
+
+Introspection is a feature of Prisma that scans your entire database and generates a `schema.prisma` file.
+This feature generates a data model that reflects your current database schema.
+
+Here's a quick overview of its main functions on SQL databases:
+
+- Map tables in the database to Prisma models 
+- Map columns in the database to the fields of Prisma models
+- Map indexes in the database to indexes in the Prisma schema  
+- Map database constraints to attributes or type modifiers in the Prisma schema
+
+To learn more see [Prisma's Introspection docs](https://www.prisma.io/docs/concepts/components/introspection#what-does-introspection-do).
+
+Run the `prisma db pull` command in the terminal to start the introspection process.
+Using this command requires your database's **Connection URL** be set in your Prisma Schema `datasource`.
```

Comment:
do we want to add instructions on how to add the connection string? or at least sent the user to a specific location on prisma's docs to understand how to do that?